extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,147 @@
+
+This extension aims to address the simplest and most common use cases for optical transparency: infinitely-thin materials with no complex scattering (e.g. dynamic scattering) or dispersion. Dealing exclusively with “thin” materials (i.e. materials where only the surface is considered and not the volume) allows many simplifications to be made when calculating things like refraction and absorption.
+
+## Extending Materials
+
+A transparent material is defined by adding the `KHR_materials_transmission` extension to any glTF material. When present, the extension indicates that a material should be rendered as a transparent surface and be blended as defined in the following spec. Alpha coverage and `doubleSided` properties still apply to transparent materials. Note that, as rendering transparent objects presents many difficult-to-solve issues with primitive ordering, this extension does not dictate rendering algorithms.
+
+```json
+materials: [
+  {
+    "extensions": {
+       "KHR_materials_transmission": {
+         "transmissionFactor": 0.8,
+         "transmissionTexture": 0
+       }
+    }
+  }
+]
+```
+
+## Properties 
+
+Only two properties are introduced with this extension and combine to describe a single value; the percentage of light that is transmitted through the surface of the material. These properties work together with the existing properties of the material to define the way light is modified as it passes through the substance. 
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**transmissionFactor** | `number` | The base percentage of light that is transmitted through the surface.| No, default:`1.0`|
+|**transmissionTexture** | [`textureInfo`] | A greyscale texture that defines the transmission percentage of the surface. This will be multiplied by transmissionFactor. | No |
+
+### transmissionFactor 
+
+The amount of light that is transmitted by the surface rather than diffusely re-emitted. This is a percentage of all the light that penetrates a surface (i.e. isn’t specularly reflected) rather than a percentage of the total light that hits a surface. A value of 1.0 means that 100% of the light that penetrates the surface is transmitted through. This is the default.
+
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>Defaulting to 1.0 allows making a material transparent by simple defining the extension.</em></figcaption>
+</figure>
+
+### transmissionTexture 
+
+A greyscale texture that defines the amount of light that is transmitted by the surface rather than diffusely re-emitted. A value of 1.0 means that 100% of the light that penetrates the surface (i.e. isn’t specularly reflected) is transmitted through. The value is linear and is multiplied by the transmissionFactor to determine the total transmission value. Note that use of transmission values does not disallow usage of alpha coverage (via `baseColor.a`).
```

Comment:
Multiplication of texture and scale value is standard for glTF - however, do we see the need to keep this.
Ie, will the texture be used with transmissionFactor != 1.0?

Reply:
If one want to use transmission without a texture, one can use the `transmissionFactor` to set the value per material. In this case, someone does not need to store the `transmissionTexture`, which does save a lot of resources.
Using the `transmissionTexture` without a factor makes sense from your side, so no need to deep dive into this case.
To have both in combination does allow to have e.g. variations. Same texture, the scalar is doing the variation. Also, if both defined, the calculation somehow needs to make sense.

In general, this is glTF style and I do not see any reason to change this. It always have been a `logical or` and there is no need to switch to an `either or`. @lexaknyazev

The effort to implement this in a shader is minimal and the effort grows linearly and not exponentially.
Finally, maybe the scalars/vectors are animated in the future.

So, at this point of time, this discussion is out of scope.

Reply:
I agree, this discussion feels out of scope here.  @rsahlin, you can detect `*Factor == 1.0` for any glTF factor at the app level, and use a shader compiler directive to get rid of the extra multiply, if you believe it's problematic.  But in practice I don't think it's something we need to shift strategy on for future extensions.

Reply:
> The effort to implement this in a shader is minimal and the effort grows linearly and not exponentially.
> Finally, maybe the scalars/vectors are animated in the future.

@UX3D-nopper My question is who uses the scale factor != 1.0 together with a texture.
Not how it is declared in the specification or how to implement in shader.

So, the answer is:
>In general, this is glTF style and I do not see any reason to change this.

That's fine!

Reply:
@emackey Yes, I do know how to handle shader variants using one super-shader :-)

>But in practice I don't think it's something we need to shift strategy on for future extensions.

Sure, that's fine.